docs/technical/api-spec.yml

```diff
@@ -17,14 +17,13 @@ paths:
         '202':
           description: Accepted
       parameters:
-        - name: X-Sent-From
+        - name: X-Sent-By
```

Comment:
Isn't 'From' a kind of standard? Why the change?

Reply:
Discussed it with Rainer and:
send to / sent by sounds better than 
send to / sent from

Reply:
Will it also be changed in DMR? https://github.com/buerokratt/DMR/blob/feature/dmr-implementation/src/Dmr.Api/Models/HeadersInput.cs

Reply:
yes, everywhere

Reply:
Possible to add the JSON schema for body for the message as well? So that it can be easily referenced.